doc/api/v8.md

```diff
@@ -135,6 +135,31 @@ to get swapped out by the operating system.
 }
 ```
 
+## v8.heapdump.getHeapdump()
+<!-- YAML
+added: REPLACEME
+-->
+
+* Returns: {Object}
+
+Returns an object containing a snapshot of the current V8 heap.
+
+## v8.heapdump.triggerHeapdump([filename])
```

Comment:
Why is it `triggerHeapdump` instead of `writeHeapdump`? (I notice that this is also the case for node-report but it does not really make a lot of sense to me since the word `trigger` does not imply writing something to disk)

Reply:
Strictly for consistency with node-report. I would have preferred write* also.

Reply:
@jasnell see https://github.com/nodejs/node/pull/26527